CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -364,9 +366,6 @@ def __init__(self, db_client: "botocore.client.DynamoDB", fridge: dict = None):
             self.verified: bool = fridge.get("verified", None)
 
     def get_item(self, fridgeId):
-        is_valid, message = Fridge.is_valid_id(fridgeId=fridgeId)
```

Comment:
if the fridgeId is not valid then it won't be in the database anyways, so this validation is redundant